aziot/src/check.rs

```diff
@@ -0,0 +1,326 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::collections::BTreeMap;
+use std::io::prelude::*;
+use std::str::FromStr;
+
+use anyhow::Result;
+use colored::Colorize;
+use structopt::StructOpt;
+
+use aziot_check_common::{
+    CheckOutputSerializable, CheckOutputSerializableStreaming, CheckResultSerializable,
+    CheckResultsSerializable,
+};
+
+use crate::internal::check::{
+    AdditionalInfo, CheckResult, CheckerCache, CheckerCfg, CheckerShared,
+};
+
+#[derive(StructOpt)]
+#[structopt(about = "Check for common config and deployment issues")]
+pub struct CheckCfg {
```

Comment:
`CheckOptions`, to be consistent with the root `Options`.

Same for `CheckListCfg`